### DISCUSSION TITLE
Self hosted - IDP with Azure - Error Managing users groups

### DISCUSSION THREAD

**Discussion by MarcosBan** [hoophq/hoop#436] [Q&A] [answered]
Hey! 

Currently im running an POC with the self hosted solution on an GKE cluster. All the components are running and the hoopgateway are integrated with Azure Entra ID - IDP as described in the docs. But im facing this error when i try to manage user groups on the portal.  "Key: 'User.Email' Error:Field validation for 'Email' failed on the 'required' tag"

Even when i try to create a new group the same error occurs. There`s an way to solve this problem?

**Comment by mtmr0x** [comment, marked as answer]
Hi, @MarcosBan , we've had some clients on Kubernetes reporting the same issue. We are working on it and will let you know when we have it solved

**Comment by mtmr0x** [comment]
Fixed on the newest version. Please upgrade to 1.23.25 (or superior for future readers 😉)

**Reply by MarcosBan** [reply]
Many thanks for the update, it`s working!
